service/controller/clusternamespace/resource/rbaccleaner/create.go

```diff
@@ -0,0 +1,45 @@
+package rbaccleaner
+
+import (
+	"context"
+
+	"github.com/giantswarm/rbac-operator/pkg/rbac"
+
+	"github.com/giantswarm/microerror"
+
+	"github.com/giantswarm/rbac-operator/service/controller/clusternamespace/key"
+)
+
+// EnsureCreated Ensures that ClusterRoleBinding and ClusterRole deployed by legacy
+// app-operators to Cluster namespaces are *deleted* when a cluster namespace is created.
```

Comment:
Do we have something similar for the org- namespaces?

Reply:
I checked and we don't have app-operators in the org- namespace in any MC. I think the move of resources to org- namespaces goes hand in hand with CAPX, so it's cluster-apps-operator running in there.

Reply:
@MarcelMue might know if I'm totally wrong or not.